swap_meet/item.py

```diff
@@ -0,0 +1,19 @@
+class Item:
+    def __init__(self, item = "Hello World!", category="", condition = 0):
+        self.category = category
+        self.item = item
+    
+    def __str__(self):
+        return f"{self.item}"
+
+    def __float__(self):
+        return f"{self.condition}"
+
+    def condition_description(self):
+        return f"{self.condition}"
```

Comment:
looks like `self.condition` is being used in `condition_description` method, but it is not assigned in the `__init__` method. Let's try something like this:

```suggestion
class Item:
    def __init__(self, category="", condition = 0):
        self.category = category
        self.condition = condition
    
    def __str__(self):
        return "Hello world!"

    def condition_description(self):
        if self.condition == 0:
            return "This item is trash!"
        # keep adding all conditions ( 0 thru 5)  to this method to give a description to the item, not just the condition number
```